java/ql/src/experimental/Security/CWE/CWE-552/UnsafeResourceGet.java

```diff
@@ -0,0 +1,16 @@
+// BAD: no URI validation
+URL url = servletContext.getResource(requestUrl);
```

Comment:
Use `request.getServletContext()` throughout the example for consistency.

```suggestion
URL url = request.getServletContext().getResource(requestUrl);
```

Reply:
Changed for consistency